test/software/amazon/ion/StructTest.java

```diff
@@ -738,6 +738,29 @@ public void testMakeNullRemovesChildsContainer()
                    val.getContainer());
     }
 
+    @Test
+    public void testMapAfterClone()
```

Comment:
Does this actually add coverage?

Reply:
It doesn't really have a way of differentiating between that internal behavior, though. If it's not adding anything, I'd vote to remove it, as there's already plenty of testing to verify that IonStructLite works correctly.